application/perfect/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM swift:latest
+
+RUN apt-get update
+
+RUN apt-get upgrade -y && \
+    apt-get install -y git
```

Comment:
`git` seems to already be installed in the base image
https://github.com/apple/swift-docker/blob/d58282b54afb6192de97d9cad973d2d2cc928dd5/5.2/ubuntu/18.04/Dockerfile#L19

Reply:
Indeed, I forgot to take it out, initially I was using a different image.
This is now fixed. Nice spot!